cost-analyzer/templates/_helpers.tpl

```diff
@@ -967,9 +967,14 @@ Begin Kubecost 2.0 templates
     - name: kubecost-enterprise-pricing
       mountPath: /var/configs/enterprise-pricing
     {{- end }}
-    {{- if and (.Values.instanceTypes.enabled) (.Values.instanceTypes.custom) }}
+    {{- if (.Values.instanceTypes).custom }}
     - name: custom-instance-types
-      mountPath: /var/configs/instance-types
+      mountPath: /var/configs/instance-types/custom
```

Comment:
Kubernetes does not allow configmap and secret to be mounted to same volumes . So im changing the volume definition to /var/configs/instance-types/custom and /var/configs/instance-types/gcp.